### PR TITLE
Add missing endpoints to README

### PR DESCRIPTION
**Pull request by andrewgaul**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -31,11 +31,12 @@ Endpoint                                 Description
 `/gzip`_                                 Returns gzip-encoded data.
 `/deflate`_                              Returns deflate-encoded data.
 `/status/:code`_                         Returns given HTTP Status code or random if more than one are given.
-`/response-headers`_                     Returns given response headers.
+`/response-headers?key=val`_             Returns given response headers.
```
Comment (sigmavirus24) [inline README.rst]: Should be `header=val`

File: README.rst [hunk 2]
```diff
@@ -31,11 +31,12 @@ Endpoint                                 Description
...
 `/redirect/:n`_                          302 Redirects *n* times.
 `/redirect-to?url=foo`_                  302 Redirects to the *foo* URL.
 `/redirect-to?url=foo&status_code=307`_  307 Redirects to the *foo* URL.
 `/relative-redirect/:n`_                 302 Relative redirects *n* times.
+`/absolute-redirect/:n`_                 302 Absolute redirects n times.
```
Comment (sigmavirus24) [inline README.rst]: Please add the appropriate emphasis in the description to match the style of the rest of this document.

File: README.rst [hunk 3]
```diff
@@ -67,12 +73,13 @@ Endpoint                                 Description
 .. _/gzip: http://httpbin.org/gzip
 .. _/deflate: http://httpbin.org/deflate
 .. _/status/:code: http://httpbin.org/status/418
-.. _/response-headers: http://httpbin.org/response-headers?Content-Type=text/plain;%20charset=UTF-8&Server=httpbin
+.. _/response-headers?key=val: http://httpbin.org/response-headers?Content-Type=text/plain;%20charset=UTF-8&Server=httpbin
```
Comment (sigmavirus24) [inline README.rst]: Also update this.

File: README.rst (hunk 1 above)
Comment (andrewgaul) [inline README.rst]: Done.

File: README.rst (hunk 3 above)
Comment (andrewgaul) [inline README.rst]: Done.

File: README.rst (hunk 2 above)
Comment (andrewgaul) [inline README.rst]: Done.

**Comment by kennethreitz** [comment]
✨🍰✨

**Pull request merged**